test/functional/p2p_add_connections.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env python3
+# Copyright (c) 2020 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+"""Test add_outbound_p2p_connection test framework functionality"""
+
+from test_framework.test_framework import BitcoinTestFramework
+from test_framework.p2p import P2PInterface
+from test_framework.util import assert_equal
+
+
+class P2PAddConnections(BitcoinTestFramework):
+    def set_test_params(self):
+        self.setup_clean_chain = False
+        self.num_nodes = 2
+
+    def skip_test_if_missing_module(self):
+        self.skip_if_no_wallet()
+
+    def check_node_connections(self, node=None, num_in=None, num_out=None):
+        info = self.nodes[node].getnetworkinfo()
+        assert_equal(info["connections_in"], num_in)
+        assert_equal(info["connections_out"], num_out)
+
+    def run_test(self):
+        self.disconnect_nodes(0, 1)
```

Comment:
Rather than connecting two nodes and then disconnecting them, you can just not have them connect to each other by overriding the `setup_network()` method:

```python
def setup_network(self):
    self.setup_nodes()
    # Don't connect the nodes
```

Reply:
👍🏽 done